emissionsapi/download.py

```diff
@@ -35,26 +65,31 @@ def download():
     date_begin = config('download', 'date', 'begin')\
         or '2019-09-10T00:00:00.000Z'
     date_end = config('download', 'date', 'end') or '2019-09-11T00:00:00.000Z'
-    countries = config('download', 'country') or ['DE']
+    countries = config('download', 'country') or None
```

Comment:
The `or None` is superfluous, isn't it?

Reply:
Yes, but I would like to leave it that way, since it kind of the explicit default here.